auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -22,6 +22,16 @@
       @input="onItemSelected($event)"
       @keydown.enter.native.prevent
     >
+      <template v-slot:append>
+        <v-progress-circular
+          v-if="state === States.SEARCHING"
+          color="primary"
+          indeterminate
+          :size="24"
+          :width="2"
+        />
+      </template>
```

Comment:
This slot is the exact size and location as the `append-icon` property above.

Icon under normal scenarios:

![image](https://github.com/bcgov/sbc-auth/assets/10002889/2242a4d7-755f-4da9-b041-3a73c416ab54)

Icon during search (it's animated so you only see an arc segment here):

![image](https://github.com/bcgov/sbc-auth/assets/10002889/ae86b088-f1f1-4eb2-93c0-0ebdd4d8c90b)